src/components/Placeholder/components/PlaceholderPrimaryAction/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactElement } from 'react';
+import clsx from 'clsx';
+import Button from '../../../Button';
+import { Props } from './types';
+
+
+function PlaceholderPrimaryAction({
+    className,
+    onClick,
+    children,
+}: Props): ReactElement {
+
+
+    const placeholderClasses = clsx(
```

Comment:
The primary and secondary actions are basically the same thing with a different class.  I would recommend considering whether this direction is better or if it might be better just to have a "variant" prop